scripts/manage-version-suffix.mjs

```diff
@@ -45,15 +50,24 @@ function addOrRemoveVersionSuffix(version) {
 }
 
 function updateVersionInNpmFile(path, newVersion) {
-  const json = fs.readFileSync(path, 'utf8').toString();
+  const json = readFileContent(path);
   const pkg = JSON.parse(json);
   pkg.version = newVersion;
   fs.writeFileSync(path, JSON.stringify(pkg, null, 2) + '\n');
 }
 
-function updateVersionInSonarFile(path, newVersion) {
-  const content = fs.readFileSync(path, 'utf8').toString();
+function updateVersionInSonarFile(newVersion) {
+  const path = './sonar-project.properties';
+  const content = readFileContent(path);
   // replace the 1st occurrence, is ok as a key appears only once in the file
   const updatedContent = content.replace(/sonar\.projectVersion=.*/, `sonar.projectVersion=${newVersion}`);
   fs.writeFileSync(path, updatedContent);
 }
+
+function updateVersionInTypeScriptVersionFile(newVersion) {
```

Comment:
I found the name of the function weird.
Without know what there is in the `src/component/version.ts` file, I though it was for the version of TypeScript, and not the version of BPMN Visu.

Reply:
`updateVersionInSourceFile`?

Reply:
Ok for me 🙂

Reply:
done in 28d45b34